regression/cbmc/CMakeLists.txt

```diff
@@ -10,16 +10,18 @@ add_test_pl_tests(
 
 add_test_pl_profile(
     "cbmc-paths-lifo"
-    "$<TARGET_FILE:cbmc> --paths lifo ${gcc_only}"
+    "$<TARGET_FILE:cbmc> --paths lifo"
```

Comment:
:+1: Yeah you’d be right @thomasspriggs, this was always the wrong place to pass in that flag and it just never became a problem before because gcc_only is empty on non-windows platforms.